### PR TITLE
Gt 443 modify onboarding screens

### PR DESCRIPTION
**Pull request by ryancarlson**
- first commit is the actual change. removes multiple buttons and therefore decision point that was confusing/annoying users
- second commit removes `self` references where they aren't necessary.

### CODE REVIEW

**Comment by ryancarlson** [comment]
@wiseguy16 can you review please?

File: godtools/ViewControllers/Platform/OnboardingViewController.swift [hunk 1]
```diff
@@ -14,35 +14,33 @@ class OnboardingViewController: BaseViewController {
     @IBOutlet weak var page2View: UIView!
     @IBOutlet weak var pageControl: UIPageControl!
     
-    @IBOutlet weak var nowButton: TransparentButton!
-    @IBOutlet weak var laterButton: TransparentButton!
-    @IBOutlet weak var okayButton: TransparentButton!
+    @IBOutlet weak var languagesOkayButton: TransparentButton!
+    @IBOutlet weak var toolsOkayButton: TransparentButton!
```
Comment (wiseguy16) [inline godtools/ViewControllers/Platform/OnboardingViewController.swift]: Don't know if these were being used in Storyboards also...just wondering if there were any "orphaned" outlets when these got switched?

Comment (ryancarlson) [reply godtools/ViewControllers/Platform/OnboardingViewController.swift]: Yeah, I went through the corresponding .xib file and made sure the appropriate references were where they needed to be.

**Comment by ryancarlson** [comment]
Thanks for the review!

**Pull request merged**